packages/transitions/src/lib/Fade.stories.tsx

```diff
@@ -0,0 +1,32 @@
+// Each exported component in the package should have its own stories file
+
+import { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import Alert from '@mui/material/Alert';
```

Comment:
I think we should use our own `Alert` here

Reply:
fixed